src/types.ts

```diff
@@ -0,0 +1,46 @@
+type Note = {
+    ATTR: any,
+    _: any,
+}
+  
+type w = {
```

Comment:
"w" is not a good name for an exported variable.  Does it need to be "w" for JSON importing, or some other reason?  If not, then I recommend naming this something that will have meaning for other developers, like "word".